src/main/java/com/checkout/tokens/CardTokenResponse.java

```diff
@@ -1,7 +1,6 @@
 package com.checkout.tokens;
```

Comment:
There is a missing property called `name`.

Reply:
Nice catch!